src/Tests/Microsoft.DotNet.GenAPI.Tests/CSharpFileBuilderTests.cs

```diff
@@ -1389,5 +1389,163 @@ public partial class foo
                     """,
                 includeInternalSymbols: false);
         }
+
+        [Fact]
+        public void TestInternalDefaultConstructorGeneration()
```

Comment:
Please add a test with `includeInternalSymbols = true`. I would expect that such a test would show that a default ctor would become a private one.

Reply:
`includeInternalSymbols` is true by default. Explicitly specify it in the `TestPrivateDefaultConstructorGeneration` test case

Reply:
Do we have a test that shows that the implicit default ctor becomes private?

Reply:
> implicit default ctor becomes private?

Could you please provide a short example?

In the example below:

```C#
public class Bar
{
/// implicit default constructor.
}
```

we won't generate `private Bar() {}` even if the `includeInternalSymbols` is set to true.